pkg/v1/tkg/test/tkgctl/shared/package.go

```diff
@@ -202,12 +212,20 @@ func checkClusterCB(ctx context.Context, mccl, wccl client.Client, mcClusterName
 		if strings.Contains(pkg.Name, "kapp-controller") {
 			verifyPackageInstall(ctx, mccl, pkg.Namespace, GeneratePackageInstallName(clusterName, pkg.Name), pkg.Spec.RefName, pkg.Spec.Version)
 		} else {
-			verifyPackageInstall(ctx, client, pkg.Namespace, GeneratePackageInstallName(clusterName, pkg.Name), pkg.Spec.RefName, pkg.Spec.Version)
+			// we skip verifying package installs in tkgs supervisor cluster
+			// in tkgs guest cluster, we skip verifying package install for capabilities package as it doesn't reconcile
+			// TODO: Verify the capabilities package install when it is fixed to reconcile
+			if infrastructureName != "tkgs" || (!isManagementCluster && pkg.Name != "capabilities") {
+				verifyPackageInstall(ctx, client, pkg.Namespace, GeneratePackageInstallName(clusterName, pkg.Name), pkg.Spec.RefName, pkg.Spec.Version)
+			}
 		}
 	}
 
-	By(fmt.Sprintf("Verify addon packages on %q cluster %q status is reflected correctly in clusterBootstrap status", clusterType, clusterName))
-	verifyPackageInstallStatusinClusterBootstrapStatus(ctx, mccl, mcClusterName, mcClusterNamespace, wcClusterName, wcClusterNamespace, isManagementCluster, packages)
+	// Skip verifying package install status in CB for tkgs guest cluster as there is no CB
```

Comment:
This is incorrect. As stated in a previous comment above, CB exists for the guest cluster and it resides in the supervisor cluster under guest cluster namespace.
You need to verify status in the CB of the guest cluster (which resides in the supervisor cluster) as well, unless it needs lots of refactoring in ```verifyPackageInstallStatusinClusterBootstrapStatus``` and the decision is to ignore that test.

Reply:
Fixed